GomokuClient/packages/gomoku-core/src/hooks/useJoinGame.ts

```diff
@@ -43,6 +47,9 @@ export const useJoinGame = (
         playerJoinedGame: async () => {
           notification.show(`You have joined the game`);
         },
+        bothPlayersJoined: async ({ players }) => {
+          setPlayers(players);
+        },
```

Comment:
Do we need event for that? If i can receive it based on second player that joined, so   playerJoinedGame event should be enough, no?

Reply:
Not really, it won't be enough, because you need to know which player is black and which player is black, when game is started and server randomly generated colors. As well, you need to send that information to spectators. If somebody visited registered players link anonymously, for example.